department-of-veterans-affairs/va.gov-cms
# VACMS-1269 homepage improvements

## Description

Relates to #1269. Builds on the last login PR, further styles the PIV login container & adds a toggle to swap between the two login flows, while staying at the same URL. Figma this is based on is here: https://www.figma.com/file/w0z1NIdY7nent0Q6r7hjTD/Login-page?node-id=330%3A64

Added a footer to the **login page only** that mimics the feel of design.va.gov and developer.va.gov. Consistency builds trust!

Still need to add styles for the "new look" block, but wasn't sure how that would be done in the current system (is there an existing block?) so left it alone for this PR

## Testing done
Lots of screen size adjustments to get spacing feeling okay on mobile/tablet in addition to the larger screens. User/pass still works when hidden... I assume PIV login still works, can't use it locally.

Fixed WAVE errors and warnings.... I looked at it with VoiceOver and with the chromevox extension and both seemed to be navigable... can probably still be improved, but it works for now.

## Screenshots
extra wide:
<img width="1917" alt="Screen Shot 2021-12-22 at 4 15 55 PM" src="https://user-images.githubusercontent.com/11279744/147169652-7858f1d2-4ccd-43b7-96db-a6f76f6fa62a.png">
<img width="1916" alt="Screen Shot 2021-12-22 at 4 16 05 PM" src="https://user-images.githubusercontent.com/11279744/147169656-01e74fa5-7544-4264-bf39-8726b7142259.png">

tablet:
<img width="788" alt="Screen Shot 2021-12-22 at 4 16 37 PM" src="https://user-images.githubusercontent.com/11279744/147169659-9d1b5c63-b18a-49ff-819d-e33626b42236.png">
<img width="788" alt="Screen Shot 2021-12-22 at 4 16 47 PM" src="https://user-images.githubusercontent.com/11279744/147169661-fcf6ca39-9f1a-47d3-af14-75fab5117398.png">


## QA steps
visit tugboat env, see that the unauth page is updated and the toggle works for switching between login flows.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?

Claro only at this point, no announcement needed.
